### DISCUSSION TITLE
Is it possible to have 2 controllers on a single instance of ZwaveJS2MQTT?

### DISCUSSION THREAD

**Discussion by miguelarios** [zwave-js/zwave-js-ui#1619] [Q&A] [answered]
Is it possible to have 2 controllers on a single instance of ZwaveJS2MQTT?

I see that S2 is now supported so I want to move away from my zstick Gen5 I have that doesn't support S2 at all to the new zstick gen7 from aeotec. However it be nice if I don't have to run two docker containers with 2 sticks to manage the migration of my devices.

**Comment by robertsLando** [comment, marked as answer]
Ii'm not sure, someone already asked this in the past, based on what I remember zwave-js doesn't support secondary controllers yet. @blhoward2 do you confirm?

**Reply by blhoward2** [reply]
Zwavejs cannot be the second controller. It's technically possible to include a second controller but for little benefit and we don't support that.
